x/evm/types/params.go

```diff
@@ -77,6 +77,49 @@ func (p Params) String() string {
 	return string(out)
 }
 
+func (p Params) MarshalJSON() ([]byte, error) {
+	alias := &struct {
+		EnableCreate                      bool   `json:"enable_create"`
+		EnableCall                        bool   `json:"enable_call"`
+		ExtraEIPs                         []int  `json:"extra_eips"`
+		EnableContractDeploymentWhitelist bool   `json:"enable_contract_deployment_whitelist"`
+		EnableContractBlockedList         bool   `json:"enable_contract_blocked_list"`
+		MaxGasLimitPerTx                  uint64 `json:"max_gas_limit_per_tx"`
+	}{
+		p.EnableCreate,
+		p.EnableCall,
+		p.ExtraEIPs,
+		p.EnableContractDeploymentWhitelist,
+		p.EnableContractBlockedList,
+		p.MaxGasLimitPerTx,
+	}
+	
+	return json.Marshal(alias)
+}
+
+func (p *Params) UnmarshalJSON(input []byte) error {
+	alias := &struct {
+		EnableCreate                      bool   `json:"enable_create"`
+		EnableCall                        bool   `json:"enable_call"`
+		ExtraEIPs                         []int  `json:"extra_eips"`
+		EnableContractDeploymentWhitelist bool   `json:"enable_contract_deployment_whitelist"`
+		EnableContractBlockedList         bool   `json:"enable_contract_blocked_list"`
+		MaxGasLimitPerTx                  uint64 `json:"max_gas_limit_per_tx"`
+	}{}
+	if err := json.Unmarshal(input, &alias); err != nil {
```

Comment:
alias is already point why put point of point in this line

Reply:
fixed